diagram.md

```diff
@@ -191,7 +191,7 @@ The `Cluster Elements` contain all of the video frames and audio for each track.
 ```
 Figure: Representation of a `Cluster Element` and its immediate `Child Elements`.
 
-Below is a representation of the Block structure.
+Below is a representation of the `Block` structure.
```

Comment:
Hmm maybe "…the `Block Element's` structure"? I don't feel too strongly about this, though.